### DISCUSSION TITLE
How to Add Plots on an external hard drive to my FARM

### DISCUSSION THREAD

**Discussion by JSNONE1** [Chia-Network/chia-blockchain#3816] [Support]
Ok, I have some plots that are on an external hard drive that were plotted by a different device, using a different wallet.

I have the recovery phrase for the "OLD" wallet.

How do I add these plots to MY farm? If that is not possible, how do I change the wallet info so that the previous owner does not have access to the wallet (assuming they saved a copy of the recovery phrase)

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
